BUILD.adoc

```diff
@@ -537,6 +540,11 @@ recent version.
     included in .gitattributes as well.
 ====
 
+Node.js is best obtained through `nvm`.
+Follow installation instructions at https://github.com/creationix/nvm#install-script .
+Then install Node.js by running `nvm install stable`.
+Verify your installation by `node --version`.
+Install necessary Node.js packages by `npm install escape-string-regexp he`.
```

Comment:
It might be better to add a `package.json`/`package-lock.json` to the repo, that way the modules get installed locally instead of globally.

Reply:
They are installed locally either way.
TBH I did not worry much about how to work with those files. I guess it might be somewhat nicer to require just `npm install` to install the dependencies...

Reply:
Added the package.json defining the dependencies.